pkg/commands/refresh.go

```diff
@@ -164,5 +213,78 @@ func refreshResultReport(newCaps, oldCaps []types.Capability) map[refreshStatus]
 			report[deleted] = append(report[deleted], oldCap)
 		}
 	}
+	_ = writeCapDefHashIntoLocal(dir, newHash)
 	return report
 }
+
+// checkRefreshInterval checks whether refresh caps from cluster or not
+// if a short time passed since last refresh, return true to use cached capabilities
+// else return false to refresh
+func checkRefreshInterval(capDir string, maxRefreshInterval time.Duration) (bool, error) {
+	timestamp := strconv.FormatInt(time.Now().Unix(), 10)
+	tmpDir := filepath.Join(capDir, ".tmp")
+	timeFilePath := filepath.Join(tmpDir, ".lastRefresh")
+	exist, _ := system.CreateIfNotExist(tmpDir)
```

Comment:
Should we delete the dir later?

Reply:
Since it's in the dir `.vela/capabilities`, I think it's okay to be deleted along with `.vela/capabilities` if necessary.